scripts/geth-allowed-packages.txt

```diff
@@ -17,4 +17,6 @@
 "github.com/ethereum/go-ethereum/ethdb/pebble"
 "github.com/ethereum/go-ethereum/event"
 "github.com/ethereum/go-ethereum/log"
+"github.com/ethereum/go-ethereum/metrics"
+"github.com/ethereum/go-ethereum/metrics.(*meterArbiter).tick"
```

Comment:
this seems a tad bit hacky, perhaps we could exclude files with `main_test.go`?
seems slightly less hacky to build the string as `"github.com/ethereum/go-ethereum/metrics" + ".(*meterArbiter).tick"` in the go file where there can be a comment explaining what's going on

Reply:
what's the requirement here and what does this do? I thought `github.com/ethereum/go-ethereum/metrics` should be enough to allow the whole package, no?

Reply:
the regex picks up "github.com/ethereum/go-ethereum/metrics.(*meterArbiter).tick" as a false positive. 
always regex's providing the entertainment factor in life.

Reply:
Done in e389c7f1adb615c132e21dccbadee5e37acb1455